.gitignore

```diff
@@ -115,4 +115,7 @@ venv.bak/
 /lib
 /7zip
 *.zip
-*.lnk
\ No newline at end of file
+*.lnk
+
+# Log files produced by import code
```

Comment:
Hmm, could this cause a problem for our trial data? the `NMEA` data comes with a `.log` suffix.